src/components/ArticleCard.jsx

```diff
@@ -7,6 +7,8 @@ export default function ArticleCard({ article }) {
       </Link>
       <br></br>
       <div className="comment_count">
+        {article.created_at.slice(0, 10)}
```

Comment:
Make sure you are putting any kind of text in a tag that represents it for accessibility such as <p>

Reply:
Just fixed. Thank you